SafeExamBrowser.SystemComponents/Registry/Registry.cs

```diff
@@ -86,6 +89,59 @@ public bool TryRead(string key, string name, out object value)
 			return success;
 		}
 
+		public bool TryGetNames(string key, out IEnumerable<string> names)
+		{
+			names = default;
+
+			RegistryKey keyObj;
+			if (!TryOpenKey(key, out keyObj))
```

Comment:
This line can be simplified by using `out var keyObj`. Also, for better readability, it would be great to not use any abbreviations or Hungarian notations et al., thus either renaming `keyObj` to `keyObject` or rather resp. better even to `registryKey`, as that appears to be the name of the framework class. Another option could be to rename the `key` parameter to `keyName` and then use `key` for the actual value in the method body below.

Reply:
And please also add the curly brackets for the body of the if-statement, again for readability and consistency reasons. There should actually be a code style or editor config file in the solution which should hint at these things, if I'm not mistaken.

Reply:
> There should actually be a code style or editor config file in the solution which should hint at these things, if I'm not mistaken.

Where can I find this exactly and how can I apply it? Seems useful

Reply:
It should be this file, I thought this would be automatically loaded by VS but I might be wrong on that: https://github.com/SafeExamBrowser/seb-win-refactoring/blob/master/.editorconfig